### PR TITLE
Remove special order handling for Private Sales

### PR DESCRIPTION
**Pull request by royliu2**

### CODE REVIEW

File: src/seaport.ts [hunk 1]
```diff
@@ -1592,15 +1592,6 @@ export class OpenSeaPort {
       throw new Error(errorMessage)
     }
 
-    // Remove fees for private orders
-    if (isPrivate) {
```
Comment (alexanderatallah) [inline src/seaport.ts:1596]: think isPrivate doesn't need to be a param to this method anymore

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 Full changelog coming soon. For now, here are the most important changes for doing major migrations:
 
+## Migrating to version 1.2
+- `OpenSeaPort::computeFees()` No longer accepts an `isPrivate` parameter.
```
Comment (royliu2) [inline CHANGELOG.md:6]: Think we should have something in the CHANGELOG for this, but not sure about new version #. Should we bump to 1.2 or 1.1.4?

Comment (sunwrobert) [reply CHANGELOG.md:6]: 1.2? i would guess

**Review by sunwrobert — APPROVED** [review]
lgtm personally

**Review by michael-cohen-eth — APPROVED** [review]

**Pull request merged**